### DISCUSSION TITLE
Balance type

### DISCUSSION THREAD

**Discussion by IMAGOFASTERTHANZOOM** [EssentialsX/Essentials#5416] [Q&A] [answered]
Hey people from EssentialsX, I wanted to ask how to change how my money looks. For example on a scoreboard instead of looking like this (10,200) I want it to look like this (10.2k)

@essentialsx

**Comment by ImDarkLaw** [comment, marked as answer]
Hey there,

We recommend using [Vault's placeholders](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#vault) when displaying the player's balance in the scoreboard.

In your case, you may want to use the **%vault_eco_balance_formatted%** placeholder. Make sure you've installed the Vault expansion using /papi ecloud download Vault if you haven't already.

**Reply by IMAGOFASTERTHANZOOM** [reply]
Thank you so much! I thought it was a setting in the config. It’s just a placeholder lol.
